connectivity/tests/pod.go

```diff
@@ -50,7 +50,8 @@ func (s *podToPod) Run(ctx context.Context, t *check.Test) {
 				continue
 			}
 			t.ForEachIPFamily(func(ipFam check.IPFamily) {
-				t.NewAction(s, fmt.Sprintf("curl-%d", i), &client, echo, ipFam).Run(func(a *check.Action) {
+				actionDesc := fmt.Sprintf("curl-%s-%d", ipFam.String(), i)
```

Comment:
nit, the following should work too:
```suggestion
				actionDesc := fmt.Sprintf("curl-%s-%d", ipFam, i)
```